django_project/base/templates/project/new_detail.html

```diff
@@ -1,6 +1,7 @@
 {% extends "project_base.html" %}
 {% load staticfiles %}
 {% load custom_markup %}
+{% load  user_perm %}
```

Comment:
I think it is better to write user_permissions instead of user_perm, to make it more clear